Song/VerseWriter.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection.Metadata;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Song
+{
+    public class VerseWriter
```

Comment:
Verse writer could use further a separate component, such as `VerseLinesWriter`, that returns first, last and middle lines. And then this component would just highlight the pattern, how a verse is composed.